scripts/fetch-asyncapi-example.js

```diff
@@ -6,7 +6,7 @@ const unzipper = require('unzipper');
 const path = require('path');
 const parser = require('@asyncapi/parser');
 
-const SPEC_EXAMPLES_ZIP_URL = 'https://github.com/asyncapi/spec/archive/refs/heads/master.zip';
+const SPEC_EXAMPLES_ZIP_URL = 'https://github.com/asyncapi/spec/archive/refs/tags/v2.2.0.zip';
```

Comment:
```suggestion
const SPEC_EXAMPLES_ZIP_URL = 'https://github.com/asyncapi/spec/archive/refs/tags/v2.3.0.zip';
```

we just released 2.3 yesterday, and long term solution for this hardcoded link will be solved with https://github.com/asyncapi/cli/issues/205

Reply:
I'm getting `Version 2.3.0 is not supported.` error at the moment.

Reply:
ah, yes, the automated version bump failed, can you bump parser 1.14?